kubelet-to-gcm/monitor/main/daemon.go

```diff
@@ -87,10 +87,10 @@ func getZone(zone string) (string, error) {
 	return zone, nil
 }
 
-// getProjectID returns projectID if it's set, and hits the GCE MD server otherwise.
+// getProjectID returns projectID if it's given, or gets it from gce if asked.
 func getProjectID(projectID string) (string, error) {
-	if projectID == "" {
-		body, err := getGCEMetaData("http://169.254.169.254/computeMetadata/v1/project/project-id")
+	if projectID == "use-gce" {
+		body, err := getGCEMetaData("http://169.254.169.254/computeMetadata/v1/project/numeric-project-id")
```

Comment:
As it is reused multiple times - can you extract the metadata server address prefix as a constant? I.e. const `metadataServerAddress = "http://169.254.169.254/computeMetadata/"` and then `getGCEMetaData(metadataServerAddress + "v1/project/..."`?